sponsorships.md

```diff
@@ -0,0 +1,61 @@
+---
+
+<h1>
+  Sponsor co.up
+  <span>Make your company a supporter of Berlin’s tech community</span>
+</h1>
+
+Interested in becoming one of co.up's supporting companies?
+
+### Why your support matters
+
+co.up is a community event space—but it’s so much more than that. It’s a place where the next generation of developers, designers, innovators and entrepreneurs are honing their craft right now; it's a place where your financial background doesn’t determine the level of support you receive; and it’s part of the spirit of the Berlin tech scene.
+
+For 10 years, co.up has provided free tech education to thousands of attendees across thousands of meetups, workshops, and learning groups. co.up kicked off the creative tech scene in Berlin when there was no infrastructure to train tech talent, no places to meet and collaborate, no places that were open and inclusive. It is safe to say that co.up has shaped a big part of the current technical work force in Berlin, and spawned many huge and influential communities in Berlin, e.g. the CSSconf and JSconf events, the OpenTechSchool movement, and more.
+
+Find a complete list of the events that take place in the space here: [https://co-up.de/events/](https://co-up.de/events/)
+
+**Now you have the chance to be a part of co.up's next chapter by becoming a co.up sponsor.** Your contribution will directly help provide an affordable space to organizations fighting to make tech more representative.
+
+**We need to raise 3000€ in monthly contributions by January 10th, 2020** in order to maintain co.up and our mission. Your company contribution will make an enormous difference in whether co.up can stay affordable and open to the community for events, learning groups, workshops, and more.
+
+### How can you sponsor co.up?
+
+You help us most by committing to a recurring [monthly contribution of at least EUR 100](https://members.co-up.de/membership_signup/new?plan_id=f7a7b200f596d0d7c77b91a6fb06339b).
+
+Alternatively, you can make a [yearly contribution of at least EUR 1200](https://members.co-up.de/membership_signup/new?plan_id=17064ed7642df9d3314c51f931ced1c5).
+
+Note: The "yearly payment" option can be cancelled immediately to limit your support to a one-time payment.
+
+### What we offer
+
+We estimate that between 20-50.000 people have come through co.up over the years, over an estimated 3000 events, and every month we currently get around 500 visitors. This is an opportunity for you to not only support the organizations in the space, but make yourself visible to the countless future talent that is currently developing their skills in our space.
+
+**These numbers are conservative best guesses, based on our calendar of the last ten years, and average visitor numbers that meetups report to us. We don’t track the exact number of visitors in the space.*
+
+Below you find some sponsoring packages that are ready to go for you. Each option is also available as a yearly payment.
```

Comment:
Can this read: Find a list of available sponsorship packages below. Each option is also available as part of a yearly payment.

Reply:
Sure, updated.